src/main/java/com/wolox/training/controllers/vo/BookVO.java

```diff
@@ -0,0 +1,64 @@
+package com.wolox.training.controllers.vo;
+
+import com.wolox.training.models.Book;
+
+public class BookVO {
```

Comment:
In this case is better use DTO, we are doing services. Also you can use Lombok for constructor and mapper object to convert to entity if is necessary

Reply:
Hi! I pushed the change. I didn't use Lombok because the last card include this feature, it's ok? Thanks!

Reply:
Perfect!